src/main/java/org/sunbird/assessment/service/AssessmentServiceImpl.java

```diff
@@ -202,4 +225,58 @@
 		return assessments;
 	}
 
+	@Override
+	public Map<String, Object> getAssessmentContent(String courseId, String assessmentContentId) {
+		Map<String, Object> result = new HashMap<>();
+		try {
+			HttpHeaders headers = new HttpHeaders();
+			HttpEntity<Object> entity = new HttpEntity<>(headers);
+
+			Object assessmentQuestionSet = cacheManager.getCache(ASSESSMENT_QNS_SET + "_" + assessmentContentId);
+
+			if (assessmentQuestionSet == null) {
+				String serviceURL = "https://igot-dev.in/api/course/v1/hierarchy/{courseId}?hierarchyType={hierarchyType}";
+				// String serviceURL = extServerProperties.getKmBaseHost()
+				// + extServerProperties.getContentHierarchyDetailEndPoint();
+				Map<String, String> params = new HashMap<>();
+				params.put("courseId", courseId);
+				params.put("hierarchyType", "detail");
+				ResponseEntity<Map> response = restTemplate.exchange(serviceURL, HttpMethod.GET, entity, Map.class,
+						params);
+
+				if (!ObjectUtils.isEmpty(response.getBody())) {
+					// get course content
+					List<Map<String, Object>> children = (List<Map<String, Object>>) ((Map<String, Object>) ((Map<String, Object>) response
+							.getBody().get(RESULT)).get("content")).get("children");
+					for (Map<String, Object> child : children) {
+						// get assessment content with id
+						if (((String) child.get("identifier")).equals(assessmentContentId)
+								&& ((String) child.get("artifactUrl")).endsWith(".json")) {
+							ResponseEntity<Map> assessmentContent = restTemplate
+									.exchange((String) child.get("artifactUrl"), HttpMethod.GET, entity, Map.class);
```

Comment:
Use constants for all the strings used in these lines... Constants should be declared in Constants.java file